site/src/pages/TemplateSettingsPage/TemplateSchedulePage/TemplateScheduleForm/TemplateScheduleForm.tsx

```diff
@@ -120,6 +132,8 @@ export const TemplateScheduleForm: FC<TemplateScheduleForm> = ({
     },
     initialTouched,
   })
+  withFormikDevtools(form)
```

Comment:
Is it still necessary? Maybe we could only call this on the dev environment like:
```ts
if(process.env.NODE_ENV === "development") {
  withFormikDevtools(form)
}
```